caikit/runtime/utils/servicer_util.py

```diff
@@ -47,8 +47,15 @@ def validate_caikit_library_class_exists(cdm, class_name):
         return getattr(cdm, class_name)
 
     except AttributeError as e:
-        log.error("<RUN24024010E>", "No Caikit Library CDM class for %s", class_name)
-        raise e
+        log.warning("<RUN24024050W>", "No Caikit Library CDM class for %s", class_name)
```

Comment:
Based on internal discussions, I'm also okay in removing the whole `validate_data_model` function altogether!

Reply:
Here's a story: https://github.com/caikit/caikit/issues/247

Reply:
Will handle in subsequent PR